client/src/actions/purchase.js

```diff
@@ -22,15 +22,19 @@ export function removeTemplate(template) {
   };
 }
 
-export function doPurchase({ onSuccess }) {
+export function doPurchase(order) {
```

Comment:
have you tested that it works with the onboarding flow now that onSuccess is removed?

Reply:
Yeah I did a run through of the whole website and it works fine.